Source/ACE/Entity/Objects/PlayerAllegiance.cs

```diff
@@ -0,0 +1,28 @@
+﻿using ACE.Common.Extensions;
+using ACE.Entity;
+using ACE.Entity.Enum;
+using ACE.Network;
+using ACE.Network.GameAction;
+using ACE.Network.GameEvent.Events;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ACE.Entity.Objects
+{
+    public partial class Player
+    {
+        [GameAction(GameActionType.AllegianceUpdateRequest)]
+        private void AllegianceUpdateRequestAction(ClientMessage message)
+        {
+            var unknown1 = message.Payload.ReadUInt32();
```

Comment:
This is bad for the Player class.  The Player class, which represents a person at the keyboard playing the game, is now responsible for parsing network Game Actions?  Please no.